### PR TITLE
Update tado.markdown

### PR DESCRIPTION
**Pull request by north3221**
Adding new service tado.set_climate_temperature_offset for pr: https://github.com/home-assistant/core/pull/45014

Also added an example for it

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/45014
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/tado.markdown [hunk 1]
```diff
@@ -163,6 +163,15 @@ You can use the service `tado.set_water_heater_timer` to set your water heater t
 | `time_period`          | no       | Time Period, Period of time the boost should last for e.g., `01:30:00` |
 | `temperature`          | yes      | String, The required target temperature e.g., `20.5`                   |
 
+### Service `tado.set_climate_temperature_offset`
+
+You can use the service `tado.set_climate_temperature_offset` to set the temprature offset for Tado climate devices 
+
+| Service data attribute | Optional | Description                                                            |
+| ---------------------- | -------- | ---------------------------------------------------------------------- |
+| `entity_id`            | no       | String, Name of entity e.g., `climate.heating`                         |
```
Comment (frenck) [inline source/_integrations/tado.markdown:172]: It is registered using `async_register_entity_service`, so entity_id became optional.

```suggestion
| `entity_id`            | yes      | String, Name of entity, e.g., `climate.heating`                       |
```

File: source/_integrations/tado.markdown [hunk 2]
```diff
@@ -180,3 +189,28 @@ script:
...
+    trigger:
+    - platform: state
+      entity_id:
+        - sensor.temp_sensor_room
+        - sensor.tado_temperature
+        
+    condition:
+    - condition: template
+      value_template: > 
+        {{ (states('sensor.tado_temperature')|float - states('sensor.temp_sensor_room')|float) > 0.5 }}
```
Comment (frenck) [inline source/_integrations/tado.markdown:207]: Can we make the template less long but using multi-line templates? So it comes readable for the user.

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by frenck** [comment]
Oh! And this needs to go into the `next` branch, as it relies on a next release.
Please change the target branch and git rebase if needed.

**Comment by north3221** [comment]
> Oh! And this needs to go into the `next` branch, as it relies on a next release.
> Please change the target branch and git rebase if needed.

Yeah that was an accident cause I didn't know what I was doing. I've changed it but cant figure out how to rebase...

**Comment by north3221** [comment]
Unless I am mistaken you cant rebase on github, so I'll close this PR and raise it again properly

**Pull request closed without merging**